_posts/2022-03-25-emergent-symbols.md

```diff
@@ -0,0 +1,708 @@
+---
+layout: post
+title: Symbolic Binding in Neural Networks through Factorized Memory Systems
+tags: [symbolic, memory, neural, binding]
+authors: Ameya Daigavane, Ansh Khurana, Shweta Bhardwaj, Gaurav Aggarwal
```

Comment:
Could you add a small abstract as an excerpt so that it renders with a nice summary in the `blog/` endpoint? This could be the abstract that you submitted in openreview, or something different if you prefer. For example:


```yaml
---
layout: post
title: Symbolic Binding in Neural Networks through Factorized Memory Systems
tags: [symbolic, memory, neural, binding]
authors: Ameya Daigavane, Ansh Khurana, Shweta Bhardwaj, Gaurav Aggarwal
excerpt: "In this blog post, we describe the paper 'Emergent Symbols through Binding in External Memory', which introduces the Emergent Symbol Binding Network (ESBN), a recurrent neural network with factorized augmented memory to solve several logical reasoning tasks. We describe the motivation and design choices of the ESBN in great detail, compare it to strong baselines, and perform a qualitative validation of the generalization claims from the original paper."
---
```

Reply:
Done!

Reply:
Thanks for adding the content! I think it needs to be under the `excerpt` tag as in the example above (not `abstract`), sorry for not making that clear

Reply:
Also make sure to not make a multi-line entry - do a single line wrapped in quotes (you can just copy my suggestion above if that's what you'd like)